vendor/engines/saas/lib/saas/null_analytics.rb

```diff
@@ -0,0 +1,10 @@
+module Saas
```

Comment:
I think this should live in: `lib/analytics/adapters` as technically its just a null adapter